### PR TITLE
[ADD] [16.0] hr_contract_update_overtime

### PR DESCRIPTION
**Pull request by Shide**
This module was developed because in certain cases, you want to
Update Overtime of your Employees.

It will be useful for you if you activate contracts module after attendance.

This module extends the functionality of Contracts and Attendances to support
recalculating Overtime and to allow you to keep up to date your Overtime.

You can Update Overtime in different ways:
- An action on the Contract History tree view
- Header button on Contract History form view
- Header button on Contract form view

MT-6583 @moduon  @rafaelbn @loida-vm @yajo @fcvalgar @Gelojr @edlopen please review if you want :)

### CODE REVIEW

**Review by loida-vm — APPROVED** [review]
Functional review done!

Perfect improvement!
Previously, overtime was counted based on the working hours indicated in the employee's record. Thanks to the schedules established in the contracts, the counting of hours was correct. But if we already had attendance before working with the contracts module, the overtime hours previously worked were not counted correctly. 
Now, thanks to this improvement, all past overtime hours are counted.

LGTM!
Thanks @Shide

**Review by fcvalgar — APPROVED** [review]
Functional review.

LGTM, thank you @Shide 

[OCA | [ADD] [16.0] hr_contract_update_overtime #170 - MT-6583 - Watch Video](https://www.loom.com/share/e9fefa2823e048d9b607192c72f914a8)

Note that the hours indicated in the report and the hours recorded for the employee may differ for the current day until the end of the employee's working time.

File: hr_contract_update_overtime/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "category": "Human Resources/Contracts",
+    "website": "https://github.com/OCA/hr-attendance",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["Shide", "rafaelbn"],
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
+        "python": ["freezegun"],
+    },
```
Comment (yajo) [inline hr_contract_update_overtime/__manifest__.py]: suggestion: remove it. Not really needed, it's already Odoo dependency.

**Review by yajo — APPROVED** [review]

**Review by nimarosa — APPROVED** [review]

**Comment by nimarosa** [comment]
Looks good. Simple and useful. 
Tested in runboat and code looks okay. 

Merging. 

/ocabot merge nobump

**Pull request merged**